controllers/activemqartemis_reconciler.go

```diff
@@ -1546,13 +1554,20 @@ func MakeContainerPorts(cr *brokerv1beta1.ActiveMQArtemis) []corev1.ContainerPor
 	containerPorts := []corev1.ContainerPort{}
 	if cr.Spec.DeploymentPlan.JolokiaAgentEnabled {
 		jolokiaContainerPort := corev1.ContainerPort{
-
 			Name:          "jolokia",
 			ContainerPort: 8778,
 			Protocol:      "TCP",
 		}
 		containerPorts = append(containerPorts, jolokiaContainerPort)
 	}
+	if cr.Spec.Console.Expose {
```

Comment:
It could be good to define the console container port even if we don't need to expose the console (only an app running in the Kubernetes cluster should be able to access a pod port), WDYT?

Reply:
Yes, a PodMonitor CR should only requires should a container port, see [Using PodMonitors](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#using-podmonitors):
```
Instead of a ServiceMonitor, we can use a PodMonitor which doesn't require the creation of a Kubernetes Service. In practice, the spec.selector label tells Prometheus which Pods should be scraped.
```

Reply:
ok, I'll just remove the condition check. Thanks Dom.